rainbow/devices/stm32.py

```diff
@@ -16,18 +16,52 @@
 #
 # Copyright 2019 Victor Servant, Ledger SAS
 
+import random
+
 import unicorn as uc
-import capstone as cs
-import pickle
 
-from binascii import hexlify
+from ..generics import rainbow_cortexm
+
+
+class rainbow_stm32(rainbow_cortexm):
+    """STM32 generic device
+
+    STMicroelectronics STM32 shares most peripherals addresses accross the family.
+    """
+
+    RNG_BASE_ADDR = 0x50060800
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Map RNG peripheral (Random Number Generator)
+        self.map_space(self.RNG_BASE_ADDR, self.RNG_BASE_ADDR + 0x12)
```

Comment:
Last register of RNG peripheral seems to be at offset 0x08, the end bound here looks wrong to me.